app/assets/src/components/views/bulk_download/BulkDownloadList.jsx

```diff
@@ -71,6 +115,28 @@ class BulkDownloadList extends React.Component {
     });
   };
 
+  onDownloadFileClick = async bulkDownload => {
+    // This should only be clickable when the bulk download has succeeded
+    // TODO(mark): Handle error case.
+    if (bulkDownload.status === "success") {
+      const outputFilePresignedUrl = await getPresignedOutputUrl(
+        bulkDownload.id
+      );
+      openUrl(outputFilePresignedUrl);
+    }
+  };
+
+  onCopyUrlClick = async bulkDownload => {
+    // This should only be clickable when the bulk download has succeeded
+    // TODO(mark): Handle error case. Want to change popup text, which involves a forceUpdate on the table.
```

Comment:
Tip: If you do not want to forceUpdate, you can always change the data value of the table. Should be seamless.

Reply:
Thanks for the tip!